src/tools/storage/s3/amazon-server-side/s3UploaderFunction/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I'm really curious on how the S3 uploader is working with node.js part around. Does it mean that we will need to add node toolchain as part of build-time dependency?

Reply:
You can find the links here
```
S3_URL=https://api.flameshot.org/
S3_CREDS_URL=https://api.img.flameshot.org/
S3_X_API_KEY=amazon-secret-key
```
from `src/tools/storage/s3/amazon-server-side/doc/config.ini.example`, I'm guessing aws lambda (nodejs runtime) is used here. They use aws lambda to upload files to aws s3.

Reply:
Yes, NodeJS is for Lambda functions, more details here:
src/tools/storage/s3/amazon-server-side/doc/README.md